eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepository.java

```diff
@@ -44,6 +44,11 @@
 
     Long countByIdsAndComplexFilters(String id, List<VariantEntityRepositoryFilter> filters);
 
+    List<VariantEntity> findByGenesAndComplexFilters(List<String> geneIds, List<VariantEntityRepositoryFilter> filters,
+                                                     Pageable pageable);
```

Comment:
A list of exclusions must be also supported for consistency with the search by region.